.github/workflows/osrm-backend.yml

```diff
@@ -55,9 +55,32 @@ jobs:
     steps:       
       - name: Check out the repo
         uses: actions/checkout@v3
+      - name: Enable osm.pbf cache
+        uses: actions/cache@v2
+        with:
+          path: berlin-latest.osm.pbf
+          key: v1-berlin-osm-pbf
+          restore-keys: |
+            v1-berlin-osm-pbf
       - name: Docker build
         run: |
-          docker build -f docker/Dockerfile . 
+          docker build -t osrm-backend-local -f docker/Dockerfile . 
+      - name: Test Docker image
+        run: |
```

Comment:
It would help to find the issue like we fixed in https://github.com/Project-OSRM/osrm-backend/pull/6304 (I checked that this job becomes red if I revert it).